tests/integration/source_test.go

```diff
@@ -14,15 +15,18 @@ var _ = Describe("odoSourceE2e", func() {
 	var context string
 	var oc helper.OcRunner
 
-	var _ = BeforeEach(func() {
+	BeforeEach(func() {
```

Comment:
Out of curiosity, what's the difference between the two forms (`var _ = BeforeEach(func()` vs `BeforeEach(func()`)? It seems to me that it just works in either case.